File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/cimrrecallresultdto/d/CiMrRecallResultDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mrqc.cimrrecallresultdto.d
{
    /// <summary>
    /// CiMrRecallResultDTO 的摘要说明。
    /// </summary>
    public class CiMrRecallResultDTO : BaseDTO
    {

        public CiMrRecallResultDTO()
        {

        }

        /// <summary>
        /// 就诊id
        /// </summary>
        public string Id_ent
        {
            get { return getAttrVal<string>("Id_ent", null); }
            set { setAttrVal<string>("Id_ent", value); }
        }

        /// <summary>
        /// 召回申请id
        /// </summary>
        public string Id_recall
        {
            get { return getAttrVal<string>("Id_recall", null); }
            set { setAttrVal<string>("Id_recall", value); }
        }

        /// <summary>
        ///  审批状态
        /// </summary>
        public string Name_state
        {
            get { return getAttrVal<string>("Name_state", null); }
            set { setAttrVal<string>("Name_state", value); }
        }

        /// <summary>
        /// 患者姓名
        /// </summary>
        public string Pat_name
        {
            get { return getAttrVal<string>("Pat_name", null); }
            set { setAttrVal<string>("Pat_name", value); }
        }

        /// <summary>
        /// 性别
        /// </summary>
        public string Pat_sex
        {
            get { return getAttrVal<string>("Pat_sex", null); }
            set { setAttrVal<string>("Pat_sex", value); }
        }

        /// <summary>
        /// 年龄
        /// </summary>
        public string Pat_age
        {
            get { return getAttrVal<string>("Pat_age", null); }
            set { setAttrVal<string>("Pat_age", value); }
        }

        /// <summary>
        /// 驳回原因
        /// </summary>
        public string Reason_reject
        {
            get { return getAttrVal<string>("Reason_reject", null); }
            set { setAttrVal<string>("Reason_reject", value); }
        }

        /// <summary>
        /// 就诊时间
        /// </summary>
        public DateTime? Dt_ent
        {
            get { return getAttrVal<FDateTime>("Dt_ent", null); }
            set { setAttrVal<FDateTime>("Dt_ent", value); }
        }

        /// <summary>
        /// 申请时间
        /// </summary>
        public DateTime? Dt_apply
        {
            get { return getAttrVal<FDateTime>("Dt_apply", null); }
            set { setAttrVal<FDateTime>("Dt_apply", value); }
        }

        /// <summary>
        /// 审批人
        /// </summary>
        public string Name_psn_approve
        {
            get { return getAttrVal<string>("Name_psn_approve", null); }
            set { setAttrVal<string>("Name_psn_approve", value); }
        }

        /// <summary>
        /// 审批时间
        /// </summary>
        public DateTime? Dt_approve
        {
            get { return getAttrVal<FDateTime>("Dt_approve", null); }
            set { setAttrVal<FDateTime>("Dt_approve", value); }
        }

        /// <summary>
        /// 召回截止时间
        /// </summary>
        public DateTime? Dt_end
        {
            get { return getAttrVal<FDate>("Dt_end", null); }
            set { setAttrVal<FDate>("Dt_end", value); }
        }

        /// <summary>
        /// 申请原因
        /// </summary>
        public string Apply_reason
        {
            get { return getAttrVal<string>("Apply_reason", null); }
            set { setAttrVal<string>("Apply_reason", value); }
        }

        /// <summary>
        /// 申请人
        /// </summary>
        public string Name_psn_apply
        {
            get { return getAttrVal<string>("Name_psn_apply", null); }
            set { setAttrVal<string>("Name_psn_apply", value); }
        }

        /// <summary>
        /// 申请科室
        /// </summary>
        public string Name_dep_apply
        {
            get { return getAttrVal<string>("Name_dep_apply", null); }
            set { setAttrVal<string>("Name_dep_apply", value); }
        }

        /// <summary>
        /// 条码号
        /// </summary>
        public string Barcode_chis
        {
            get { return getAttrVal<string>("Barcode_chis", null); }
            set { setAttrVal<string>("Barcode_chis", value); }
        }

        /// <summary>
        /// 就诊类型
        /// </summary>
        public string Code_entp
        {
            get { return getAttrVal<string>("Code_entp", null); }
            set { setAttrVal<string>("Code_entp", value); }
        }

        /// <summary>
        /// 是否超时
        /// </summary>
        public string Fg_timeout
        {
            get { return getAttrVal<string>("Fg_timeout", null); }
            set { setAttrVal<string>("Fg_timeout", value); }
        }

        /// <summary>
        /// 就诊编码
        /// </summary>
        public string Ent_code
        {
            get { return getAttrVal<string>("Ent_code", null); }
            set { setAttrVal<string>("Ent_code", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[] { "Id_ent", "Id_recall", "Name_state", "Pat_name", "Pat_sex", "Pat_age", "Reason_reject", "Dt_ent", "Dt_apply", "Name_psn_approve", "Dt_approve", "Dt_end", "Apply_reason", "Name_psn_apply", "Name_dep_apply", "Barcode_chis", "Code_entp", "Fg_timeout", "Ent_code" };
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mrqc.cimrrecallresultdto.d.CiMrRecallResultDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/cimrrecallresultdto/d/CiMrRecallResultDTO.java
package iih.ci.mrqc.cimrrecallresultdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 门诊病历召回列表 DTO数据 
 * 
 */
public class CiMrRecallResultDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 就诊id
	 * @return String
	 */
	public String getId_ent() {
		return ((String) getAttrVal("Id_ent"));
	}
	/**
	 * 就诊id
	 * @param Id_ent
	 */
	public void setId_ent(String Id_ent) {
		setAttrVal("Id_ent", Id_ent);
	}
	/**
	 * 召回申请id
	 * @return String
	 */
	public String getId_recall() {
		return ((String) getAttrVal("Id_recall"));
	}
	/**
	 * 召回申请id
	 * @param Id_recall
	 */
	public void setId_recall(String Id_recall) {
		setAttrVal("Id_recall", Id_recall);
	}
	/**
	 *  审批状态
	 * @return String
	 */
	public String getName_state() {
		return ((String) getAttrVal("Name_state"));
	}
	/**
	 *  审批状态
	 * @param Name_state
	 */
	public void setName_state(String Name_state) {
		setAttrVal("Name_state", Name_state);
	}
	/**
	 * 患者姓名
	 * @return String
	 */
	public String getPat_name() {
		return ((String) getAttrVal("Pat_name"));
	}
	/**
	 * 患者姓名
	 * @param Pat_name
	 */
	public void setPat_name(String Pat_name) {
		setAttrVal("Pat_name", Pat_name);
	}
	/**
	 * 性别
	 * @return String
	 */
	public String getPat_sex() {
		return ((String) getAttrVal("Pat_sex"));
	}
	/**
	 * 性别
	 * @param Pat_sex
	 */
	public void setPat_sex(String Pat_sex) {
		setAttrVal("Pat_sex", Pat_sex);
	}
	/**
	 * 年龄
	 * @return String
	 */
	public String getPat_age() {
		return ((String) getAttrVal("Pat_age"));
	}
	/**
	 * 年龄
	 * @param Pat_age
	 */
	public void setPat_age(String Pat_age) {
		setAttrVal("Pat_age", Pat_age);
	}
	/**
	 * 驳回原因
	 * @return String
	 */
	public String getReason_reject() {
		return ((String) getAttrVal("Reason_reject"));
	}
	/**
	 * 驳回原因
	 * @param Reason_reject
	 */
	public void setReason_reject(String Reason_reject) {
		setAttrVal("Reason_reject", Reason_reject);
	}
	/**
	 * 就诊时间
	 * @return FDateTime
	 */
	public FDateTime getDt_ent() {
		return ((FDateTime) getAttrVal("Dt_ent"));
	}
	/**
	 * 就诊时间
	 * @param Dt_ent
	 */
	public void setDt_ent(FDateTime Dt_ent) {
		setAttrVal("Dt_ent", Dt_ent);
	}
	/**
	 * 申请时间
	 * @return FDateTime
	 */
	public FDateTime getDt_apply() {
		return ((FDateTime) getAttrVal("Dt_apply"));
	}
	/**
	 * 申请时间
	 * @param Dt_apply
	 */
	public void setDt_apply(FDateTime Dt_apply) {
		setAttrVal("Dt_apply", Dt_apply);
	}
	/**
	 * 审批人
	 * @return String
	 */
	public String getName_psn_approve() {
		return ((String) getAttrVal("Name_psn_approve"));
	}
	/**
	 * 审批人
	 * @param Name_psn_approve
	 */
	public void setName_psn_approve(String Name_psn_approve) {
		setAttrVal("Name_psn_approve", Name_psn_approve);
	}
	/**
	 * 审批时间
	 * @return FDateTime
	 */
	public FDateTime getDt_approve() {
		return ((FDateTime) getAttrVal("Dt_approve"));
	}
	/**
	 * 审批时间
	 * @param Dt_approve
	 */
	public void setDt_approve(FDateTime Dt_approve) {
		setAttrVal("Dt_approve", Dt_approve);
	}
	/**
	 * 召回截止时间
	 * @return FDate
	 */
	public FDate getDt_end() {
		return ((FDate) getAttrVal("Dt_end"));
	}
	/**
	 * 召回截止时间
	 * @param Dt_end
	 */
	public void setDt_end(FDate Dt_end) {
		setAttrVal("Dt_end", Dt_end);
	}
	/**
	 * 申请原因
	 * @return String
	 */
	public String getApply_reason() {
		return ((String) getAttrVal("Apply_reason"));
	}
	/**
	 * 申请原因
	 * @param Apply_reason
	 */
	public void setApply_reason(String Apply_reason) {
		setAttrVal("Apply_reason", Apply_reason);
	}
	/**
	 * 申请人
	 * @return String
	 */
	public String getName_psn_apply() {
		return ((String) getAttrVal("Name_psn_apply"));
	}
	/**
	 * 申请人
	 * @param Name_psn_apply
	 */
	public void setName_psn_apply(String Name_psn_apply) {
		setAttrVal("Name_psn_apply", Name_psn_apply);
	}
	/**
	 * 申请科室
	 * @return String
	 */
	public String getName_dep_apply() {
		return ((String) getAttrVal("Name_dep_apply"));
	}
	/**
	 * 申请科室
	 * @param Name_dep_apply
	 */
	public void setName_dep_apply(String Name_dep_apply) {
		setAttrVal("Name_dep_apply", Name_dep_apply);
	}
	/**
	 * 条码号
	 * @return String
	 */
	public String getBarcode_chis() {
		return ((String) getAttrVal("Barcode_chis"));
	}
	/**
	 * 条码号
	 * @param Barcode_chis
	 */
	public void setBarcode_chis(String Barcode_chis) {
		setAttrVal("Barcode_chis", Barcode_chis);
	}
	/**
	 * 就诊类型
	 * @return String
	 */
	public String getCode_entp() {
		return ((String) getAttrVal("Code_entp"));
	}
	/**
	 * 就诊类型
	 * @param Code_entp
	 */
	public void setCode_entp(String Code_entp) {
		setAttrVal("Code_entp", Code_entp);
	}
	/**
	 * 是否超时
	 * @return String
	 */
	public String getFg_timeout() {
		return ((String) getAttrVal("Fg_timeout"));
	}
	/**
	 * 是否超时
	 * @param Fg_timeout
	 */
	public void setFg_timeout(String Fg_timeout) {
		setAttrVal("Fg_timeout", Fg_timeout);
	}
	/**
	 * 就诊编码
	 * @return String
	 */
	public String getEnt_code() {
		return ((String) getAttrVal("Ent_code"));
	}
	/**
	 * 就诊编码
	 * @param Ent_code
	 */
	public void setEnt_code(String Ent_code) {
		setAttrVal("Ent_code", Ent_code);
	}
}